qm_selinux.8

```diff
@@ -0,0 +1,345 @@
+.TH  "qm_selinux"  "8"  "23-03-21" "qm" "SELinux Policy qm"
```

Comment:
I've committed this because I'm unable to generate it in rpmbuild as it needs sudo access. Let me know if there's a way around it.

Also, the container-selinux repo does have `container-selinux.8` committed into the repo itself.

Reply:
Ok, you are right. I am thinking this was like podman man pages, but this man page is generated from the policy.